### ISSUE TITLE
Add medplum version to <meta> tag in app

### ISSUE THREAD

**Issue by rahul1** [medplum/medplum#2566]
To help debug self-hosted deploys, we should add the the version string to a `<meta>` tag

**Comment by codyebberson** [comment]
See how we construct the `MEDPLUM_VERSION` constant in `core`: https://github.com/medplum/medplum/blob/main/packages/core/esbuild.mjs#L10

See Vite HTML Env Replacement: https://vitejs.dev/guide/env-and-mode.html#html-env-replacement

So this will require adding a line to `packages/app/index.html` such as:

```html
<meta name="medplum-version" content="%MEDPLUM_VERSION%" />
```

And some setup code to `packages/app/vite.config.ts` such as:

```ts
const gitHash = execSync('git rev-parse --short HEAD').toString().trim();
process.env.MEDPLUM_VERSION = packageJson.version + '-' + gitHash;
```

**Comment by kevinlu1248** [comment]
@codyebberson Can Sweep take on this issue?

**Comment by codyebberson** [comment]
Fixed in https://github.com/medplum/medplum/pull/2692

Thanks @charlesbthomas !

**Comment by codyebberson** [comment]
> @codyebberson Can Sweep take on this issue?

Ack, sorry @kevinlu1248, missed your message!

**Comment by kevinlu1248** [comment]
@codyebberson No worries. I noticed you haven't been using Sweep recently, are there things we could build that would support Medplum better? I also noticed previously you were getting Sweep to edit small frontend changes like adding the JSX key param and writing unit tests, how was your experience with that?

We've shipped a lot of new features oriented towards JSX/TSX stacks like formatters, linters and migration-focussed tools. Would love to chat more over a call to set those up for you and to chat about your experience https://calendly.com/kevin-lu-sweep/quick-15-min.
